app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -512,9 +517,7 @@ private void updateSchedule() {
         getSchedule.execute();
     }
 
-    /**
-     * Return to previous activity on the stack,
-     */
+
```

Comment:
This line is unnecessary. Remove it. You should format your code before pushing. To use our agreed code style, please see this (https://github.com/square/java-code-styles), install, set-up (select SquareAndroid in the preferred style) and format all of the classes you modify after you set it up